setup.js

```diff
@@ -4,7 +4,7 @@ const voucherify = require("voucherify")({
   applicationId: process.env.APPLICATION_ID,
   clientSecretKey: process.env.CLIENT_SECRET_KEY,
 });
-const campaigns = require("./campaigns");
+const campaigns = require("./src/campaigns");
```

Comment:
right now `src/campaigns.js` makes me think that it's a regular piece of our code, but in reality it's only intended to be used by this setup - so why don't we just inline it here?

Reply:
You're 100% right